src/material-icons.js

```diff
@@ -3,18 +3,27 @@ import shapes from './shapes';
 
 const { number, object, string } = PropTypes;
 
-export class Icon extends Component {
+class Icon extends Component {
```

Comment:
It was already exported at the bottom of the file, and there was an error or warning in the compile process complaining it had already been exported. I tested this change in the JDL and it did not break anything.